# BIOC3301
All batch codes used in project report data analysis.

Standard workflow:
Mapping file validation.
Join paired reads.
Demultiplex.
OTU picking - option to use SILVA reference database with open reference UCLUST clustering not available, hence manual assignment of taxonomy and formation of OTU table (OTU table is read to obtain sampling depth (-e) for diversity analysis).
Diversity analysis - for alpha and beta diversity metrics.
UPGMA phylogenetic tree.
Statistical tests - ANOSIM (-c = UPGMA clustering), BIO-ENV (-c = UPGMA clustering and longitude) and db-RDA (-c = distance).

Merged 15/16, 16/17 & 17/18 datasets workflow:
Concatenated mapping file validation.
Join paired reads - 17/18 only, due to poor quality Read2 in 15/16 and 16/17 datasets.
Demultiplex - run independently for each dataset.
OTU picking - run independently for each dataset, closed reference UCLUST clustering with SILVA 128 reference database (timing errors restricted use of open reference).
OTU tables merged - OTU table is read to obtain sampling depth (-e) for diversity analysis.
Diversity analysis - for alpha and beta diversity metrics.
UPGMA phylogenetic tree.
Statistical tests - ANOSIM (-c = UPGMA clustering, to test significance clades), (-c = rainfall, to test significance of presence/absence of rain), (-c = rainfallmm, to test significance of the amount of rain).

GreenGenes workflow:
Mapping file validation.
Join paired reads - same as standard workflow.
Demultiplex - same as standard workflow.
OTU picking - open reference UCLUST clustering with GreenGenes 13_8 reference database, timing error hence manual sequence alignment, taxonomy assignment, phylogenetic tree formation and OTU table formation (with OTU table read to obtain sampling depth (-e) for diversity analysis).
Diversity analysis - for alpha and beta diversity metrics.

Stringent workflow:
Mapping file validation.
Join paired reads - stringent settings used, -j (minimum allowed overlap) = 80 and -p (maximum allowed percentage difference) = 80.
Demultiplex - stringent settings used, -q = 19.
OTU picking - option to use SILVA reference database with open reference UCLUST clustering not available, hence manual assignment of taxonomy (with stringent setting, --similarity 0.97 (minimum percentage similarity for a match)) and formation of OTU table (OTU table is read to obtain sampling depth (-e) for diversity analysis).
Diversity analysis - for alpha and beta diversity metrics.
